tmva/sofie/inc/TMVA/ROperator_Max.hxx

```diff
@@ -0,0 +1,87 @@
+#ifndef TMVA_SOFIE_ROPERATOR_MAX
+#define TMVA_SOFIE_ROPERATOR_MAX
+
+#include "TMVA/SOFIE_common.hxx"
+#include "TMVA/ROperator.hxx"
+#include "TMVA/RModel.hxx"
+
+#include <sstream>
+
+namespace TMVA{
+namespace Experimental{
+namespace SOFIE{
+
+template <typename T>
+class ROperator_Max final : public ROperator
+{
+
+private:
+
+   std::string fNX1;
+   std::string fNX2;
+   std::string fNY;
+   std::vector<size_t> fShape;
+
+public:
+   ROperator_Max(){}
+   ROperator_Max(std::string nameX1, std::string nameX2, std::string nameY):
```

Comment:
The signature of the constructor for `ROperator_Max` should be 
`ROperator_Max( const std::vector<std::string> & inputNames, std::string nameY)`